pyproject.toml

```diff
@@ -49,12 +49,19 @@ requires = ["hatchling"]
 build-backend = "hatchling.build"
 
 [tool.hatch.build.targets.sdist]
-include = ["src/*", "docs/releases.rst", "LICENSE"]
+include = [
+    "/src",
+    "/docs",
+    "/tests",
+]
 sources = ["src"]
 
 [tool.hatch.build.targets.wheel]
-include = ["src/*", "docs/releases.rst", "LICENSE"]
+include = [
```

Comment:
I'm not in front of a computer right now to check but I don't think you need the `include` here, `packages` should suffice.